test/source/tests/decrypt.ts

```diff
@@ -1318,5 +1318,48 @@ d6Z36//MsmczN00Wd60t9T+qyLz0T4/UG2Y9lgf367f3d+kYPE0LS7mXuFmjlPXfw0nKyVsSeFiu
         });
       })
     );
+
+    test(
+      'settings - download batch file attachment (should show a warning message)',
+      testWithBrowser('compatibility', async (t, browser) => {
+        const threadId = '187365d19ec9a10c';
+        const threadId2 = '18736a0687a8426b';
+        const acctEmail = 'flowcrypt.compatibility@gmail.com';
+        const expectedErrMsg = 'This executable file was not checked for viruses, and may be dangerous to download or run. Proceed anyway?';
+        const inboxPage = await browser.newExtensionPage(t, `chrome/settings/inbox/inbox.htm?acctEmail=${acctEmail}&threadId=${threadId}`);
+        const pgpBlockPage = await inboxPage.getFrame(['pgp_block.htm']);
+        await pgpBlockPage.waitAndClick('@download-attachment-0'); // test for inline attachment
+        await inboxPage.waitAndRespondToModal('confirm', 'confirm', expectedErrMsg);
```

Comment:
need to test if the file is downloaded by using `awaitDownloadTriggeredByClicking`

Reply:
done - thanks!